proto/decentraland/social_service/v2/social_service_v2.proto

```diff
@@ -425,6 +425,78 @@ message GetIncomingPrivateVoiceChatRequestResponse {
   }
 }
 
+// Community Voice Chat messages
+
+// Community voice chat credentials - specific type for community chats
+message CommunityVoiceChatCredentials {
+  string connection_url = 1;
+}
+
+// Starting a community voice chat
+message StartCommunityVoiceChatPayload {
+  string community_id = 1;
+}
+
+message StartCommunityVoiceChatResponse {
+  message Ok {
+    string voice_chat_id = 1;
```

Comment:
We can either:
1. Have different rooms, one per voice chat initiated
2. Have a single room, with the same id o room name for each community voice chat, whose id is related to the community.

If we go with the second option, the `voice_chat_id` should not be needed.
The same happens with the `JoinCommunityVoiceChatResponse` type.